sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2770,7 +2783,7 @@ Accept wildcard characters: False
 ```
 
 ### -ExpireVersionsAfterDays
-When Version History Limits are managed Manually (EnableAutoExpirationVersionTrim $false), admins will need to set the limits to the number of major versions (MajorVersionLimit) and the time period the versions are stored (ExpireVersionsAfterDays). Please check the description of EnableAutoExpirationVersionTrim for more details.
+When version history limits are managed manually (`EnableAutoExpirationVersionTrim $false`), admins will need to set the limits to the number of major versions (`MajorVersionLimit`) and the time period the versions are stored (`ExpireVersionsAfterDays`). Please check the description of `EnableAutoExpirationVersionTrim` for more details.
```

Comment:
The valid values are:

- True - When set to true and feature roll out to your tenant has completed, admin version history controls at organization, site and library levels will be available. 
- False (default) - When set to false, the feature will be disabled for your tenant. 
 
Note: Disabling the feature after previously enabling it, does not revert changes made when the feature was enabled.

Reply:
Should for parameter "EnableVersionExpirationSetting"